fahrschein/src/test/java/org/zalando/fahrschein/NakadiClientTest.java

```diff
@@ -273,15 +274,15 @@ public void shouldHandleMultiStatusWhenPublishing() throws IOException {
         server.expectRequestTo("http://example.com/event-types/foobar/events", "POST")
                 .andExpectJsonPath("$[0].id", equalTo("1"))
                 .andExpectJsonPath("$[1].id", equalTo("2"))
-                .andRespondWith(207, ContentType.APPLICATION_JSON, "[{\"publishing_status\":\"failed\",\"step\":\"validating\",\"detail\":\"baz\"}]")
+                .andRespondWith(207, ContentType.APPLICATION_JSON, "[{\"eid\":\"event-one\",\"publishing_status\":\"failed\",\"step\":\"validating\",\"detail\":\"baz\"}]")
                 .setup();
 
-        expectedException.expect(EventPublishingException.class);
-        expectedException.expectMessage("returned status [failed] in step [validating] with detail [baz]");
```

Comment:
Test failed after migration to JUnit5: Apparently, these checks were not actually executed in JUnit4.